src/lib/structures/Command.ts

```diff
@@ -799,6 +805,8 @@ export namespace Command {
 	export type JSON = CommandJSON;
 	export type Context = AliasPiece.Context;
 	export type RunInTypes = CommandOptionsRunType;
+	export type Interaction = CommandInteraction;
```

Comment:
Sike. Not correct! You should export `ChatInputInteraction`, `ContextMenuInteraction` and `AutocompleteInteraction` separately